### PR TITLE
Add R1: Recommend spack smoke testing

### PR DESCRIPTION
**Pull request by shuds13**
- [x] Review xSDK compatibility policies.
- [ ] Compare with E4S policies.
- [x] Decide any new policies or policy changes.
- [x] Also, whether any recommended policies should become mandatory.
- [ ] Ensure, `package_policies/README.md` is up to date

Also see #16

### CODE REVIEW

**Comment by shuds13** [comment]
For R9 I will add that `spack test run <package>` should work following installation to give pass/failure.

**Comment by balay** [comment]
BTW:  needs an entry in `package_policies/README.md` aswell

**Review by ulrikeyang — APPROVED** [review]
This looks fine assuming R9 is moved to R1.

File: package_policies/R9.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+**R9.** Each package should have at least one validation (smoke) test that can be invoked through the Spack package.  This will be a post-installation test that validates the usability of the package and must be executable via `spack test run <package_name>`.
+
+Policy compliance requires that the `spack test run` interface works on a standalone system. For MPI tests, MPICH or OpenMPI may be used.
```
Comment (mgates3) [inline package_policies/R9.md]: They prefer "Open MPI", with a space. https://www.open-mpi.org/

**Review by mgates3 — APPROVED** [review]
Other than the minor "Open MPI" fix, looks good.

**Review by xiaoyeli — APPROVED** [review]

**Review by gardner48 — APPROVED** [review]

**Review by joseeroman — APPROVED** [review]

File: package_policies/R9.md (hunk 1 above)
Comment (shuds13) [reply package_policies/R9.md]: Fixed

**Review by balay — APPROVED** [review]

**Review by stomov — APPROVED** [review]

**Review by hartwiganzt — APPROVED** [review]

**Review by luszczek — APPROVED** [review]

**Review by v-dobrev — APPROVED** [review]

**Pull request merged**